db/state/domain.go

```diff
@@ -162,6 +162,14 @@ func (d *Domain) kvBtAccessorFilePathMask(fromStep, toStep kv.Step) string {
 	return filepath.Join(d.dirs.SnapDomain, fmt.Sprintf("*-%s.%d-%d.bt", d.FilenameBase, fromStep, toStep))
 }
 
+func (d *Domain) firstTxNumInDB(tx kv.Tx) (firstTxNum uint64, found bool) {
```

Comment:
this method is not about `Domain` - it's about `History`. So, better move it to `History` class.